Makefile

```diff
@@ -1,86 +1,31 @@
-
-all: java-8 java-8-appdynamics java-10 java-10-appdynamics java-11 java-11-appdynamics java-12 java-12-appdynamics java-13 java-13-appdynamics java-14 java-14-appdynamics java-15 java-15-appdynamics java-16 java-16-appdynamics wildfly-17 node-express-9 node-express-12 node-express-14 $(python_targets)
-
-java-common:
-	docker build -t navikt/java:common java-common
-
-java-8: java-common
-	docker pull openjdk:8u242-jdk
-	docker build -t navikt/java:8 java-8
-java-8-appdynamics: java-common
-	docker pull openjdk:8u242-jdk
-	docker build -t navikt/java:8-appdynamics java-8/appdynamics
-
-java-10: java-common
-	docker pull openjdk:10-jdk-slim
-	docker build -t navikt/java:10 java-10
-java-10-appdynamics: java-common
-	docker pull openjdk:10-jdk-slim
-	docker build -t navikt/java:10-appdynamics java-10/appdynamics
+JAVA_VERSIONS = 8 9 10 11 13 14 15 16
+java_targets = $(addprefix java-, $(JAVA_VERSIONS))
 
-java-11: java-common
-	docker pull openjdk:11-jdk-slim
-	docker build -t navikt/java:11 java-11
-java-11-appdynamics: java-common
-	docker pull openjdk:11-jdk-slim
-	docker build -t navikt/java:11-appdynamics java-11/appdynamics
+NODE_VERSIONS = 9 12 14
+node_targets = $(addprefix node-, $(NODE_VERSIONS))
 
-java-12: java-common
-	docker pull adoptopenjdk/openjdk12:slim
-	docker build -t navikt/java:12 java-12
-java-12-appdynamics: java-common
-	docker pull adoptopenjdk/openjdk12:slim
-	docker build -t navikt/java:12-appdynamics java-12/appdynamics
+.PHONY: all common $(java_targets) $(node_targets) $(python_targets) wildfly-17
 
-java-13: java-common
-	docker pull adoptopenjdk/openjdk13:slim
-	docker build -t navikt/java:13 java-13
-java-13-appdynamics: java-common
-	docker pull adoptopenjdk/openjdk13:slim
-	docker build -t navikt/java:13-appdynamics java-13/appdynamics
-
-java-14: java-common
-	docker pull adoptopenjdk/openjdk14:slim
-	docker build -t navikt/java:14 java-14
-java-14-appdynamics: java-common
-	docker pull adoptopenjdk/openjdk14:slim
-	docker build -t navikt/java:14-appdynamics java-14/appdynamics
-
-java-15: java-common
-	docker pull openjdk:15-slim
-	docker build -t navikt/java:15 java-15
-java-15-appdynamics: java-common
-	docker pull openjdk:15-slim
-	docker build -t navikt/java:15-appdynamics java-15/appdynamics
-
-java-16: java-common
-	docker pull openjdk:16-slim
-	docker build -t navikt/java:16 java-16
-java-16-appdynamics: java-common
-	docker pull openjdk:16-slim
-	docker build -t navikt/java:16-appdynamics java-16/appdynamics
+all: $(java_targets) $(node_targets) $(python_targets) wildfly-17
 
 wildfly-17:
 	docker build -t navikt/wildfly:17 wildfly-17
 
 common:
 	docker build -t navikt/common:0.2 common
 
-node-express-9: common
-	docker pull node:9-alpine
-	docker build -t navikt/node-express:9-common node-express-9
-node-express-12: common
-	docker pull node:12-alpine
-	docker build -t navikt/node-express:12.18-alpine node-express-12
```

Comment:
De som depender på `navikt/node-express:12.18-alpine` vil nå bli "stuck" der, burde vi gjort noe grep for å fange opp disse?

Reply:
Kan ta et raskt søk gjennom github-orgen for å se hvor mange det evt er snakk om.

Reply:
Finner 5 stk, vi kan om så ta kontakt med hver av dem direkte.

Reply:
Det kan vært lurt. :+1: